g6/priceString.yml

```diff
@@ -16,10 +16,10 @@ validations:
     message: 'Pricing unit requires an answer. If none of the provided units apply, please choose ''Unit''.'
   -
     name: min_price_not_a_number
-    message: 'Prices must be numbers only, without units, eg 99.95'
+    message: 'Minimum price must be a number, without units, eg 99.95'
   -
     name: max_price_not_a_number
-    message: 'Prices must be numbers only, without units, eg 99.95'
+    message: 'Maximum must be a number, without units, eg 99.95'
```

Comment:
Why just "Maximum must..." here but "Minimum price must.." above?  Might be better with "Maximum price must..." too.